.github/workflows/nightly-check-db.yml

```diff
@@ -0,0 +1,76 @@
+name: Nightly check db
+
+on: [push]
+  # schedule:
+  #   - cron: '0 3 * * *'
+
+env:
+  PYTHONPATH: ${{ github.workspace }}
+
+jobs:
+  download-snapshaot:
+
+    runs-on: ubuntu-latest
+    container: quarkchaindocker/pyquarkchain:mainnet1.1.1
+    
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Download db snapshot
+      env:
+        PYTHONPATH: /tmp
+      run: |
+        mkdir -p workspace
+        curl -s https://s3-us-west-2.amazonaws.com/qkcmainnet/data/sample-data.tar.gz --output data.tar.gz
+        tar xvfz data.tar.gz
+        rm data.tar.gz && mv data workspace/
+
+    - name: Save cache
```

Comment:
can skip saving/loading cache and use a single job

Reply:
Done.